pandas/io/json/_normalize.py

```diff
@@ -26,12 +27,42 @@ def convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
+def _parse_use_keys(use_keys: Union[str, List, Callable, None]) -> Callable:
+    """
+    Converts different types of use_keys into a callable
+
+    Parameters
+    ----------
+    use_keys : Str, List or Callable
+        Returns true or false depending on whether to include or exclude a key
+
+        .. versionadded:: 1.0.0
+
+    Returns
+    -------
+    use_key - Callable
+     It Decides on whether to include a key in processing.
+
+    """
+    if callable(use_keys):
+        return use_keys
+
+    if is_list_like(use_keys):
+        return lambda x: x in use_keys  # type: ignore
+
+    if isinstance(use_keys, str):
+        return lambda x: x == use_keys
+
+    return lambda x: True
```

Comment:
I think we should raise a `TypeError` is `use_keys` is something like an int, or a dict